qa/tasks/deepsea.py

```diff
@@ -75,17 +75,21 @@ def _check_config_key(key, default_value):
             if key not in config or not config[key]:
                 config[key] = default_value
 
-        _check_config_key('repo', 'https://github.com/SUSE/DeepSea.git')
+        _check_config_key('repo', '')
```

Comment:
This changes the default from `https://github.com/SUSE/DeepSea.git` to `''` - need to check what the implications of this are.

Reply:
Seems fine, since it is checked further down. Personally I'd use something like 
```
_check_config_key('repo', None)
if self.config["repo"]:
// has a repo url
else:
// use package
```
But thats nit picking at best.

Reply:
@jan--f  Actually I tried the None value first, the 'None' way does not work as expected because the key will be ignored when merging overrides. This is why I had to use '' (empty string).